tendrl/commons/flows/base_flow.py

```diff
@@ -156,18 +156,19 @@ def extract_atom_details(self, atom_name):
         namespace = atom_name.split('.objects.')[0]
         object_name = atom_name.split('.objects.')[1].split('.atoms.')[0]
         atoms = self.definitions[namespace]['objects'][object_name]['atoms']
-        atom = atoms[atom_name.split('.')[-1]]
+        atom = atoms[atom_name.split('.')[-2]]
```

Comment:
Please check https://github.com/Tendrl/node-agent/pull/147 for final naming. The problem needs to be tackled at both the source code side and the Definitions side.

The above PR will clear the confusion, keep an eye